docs/docs/alephium/types.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Types
```

Comment:
Maybe we should also mention about `U512` in this section?

Reply:
We could, but it doesn't relate to the SDK. Mentioning U512 isn't really necessary either, since it doesn't appear in the interface but is used internally.

Let's postpone adding this to a later date